app/MMB/src/list/MulticityFlight.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from 'react-relay';
+import idx from 'idx';
+
+import CityImage from './CityImage';
+import type { MulticityFlight_booking as MulticityFlightType } from './__generated__/MulticityFlight_booking.graphql';
+
+type Props = {|
+  imageUrl: string,
+  type: string,
+  passengerCount: number,
+  booking: MulticityFlightType,
+|};
+
+const MulticityFlight = (props: Props) => (
+  <CityImage
+    imageUrl={props.imageUrl}
+    type={props.type}
+    passengerCount={props.passengerCount}
+    bookingId={idx(props.booking, _ => _.databaseId)}
+    status={idx(props.booking, _ => _.status) || ''}
+    departureCity={idx(props.booking, _ => _.start.airport.city.name) || ''}
+    date={idx(props.booking, _ => _.start.time)}
+    arrivalCity={idx(props.booking, _ => _.end.airport.city.name) || ''}
+  />
```

Comment:
Great simplification would be to allow `CityImage` to fetch data (create fragment). But maybe you want it pure - that's fine. In this case, you can have `CityImage` and `CityImageContainer` (fragment itself). The benefit of this approach is that you'll do just `...CityImage_data` in every component (oneway/multicity/return). Please think about it during your following refactorings...